.doctrine-project.json

```diff
@@ -0,0 +1,18 @@
+{
+    "active": true,
+    "name": "PHPCR Bundle",
+    "slug": "phpcr-bundle",
+    "docsSlug": "doctrine-phpcr-bundle",
+    "versions": [
+        {
+            "name": "3.x",
+            "branchName": "3.x",
+            "slug": "latest",
+            "current": true,
+            "aliases": [
+                "current",
+                "stable"
+            ]
```

Comment:
I don't  think this is needed: https://github.com/doctrine/doctrine-website/blob/4c29e4049bc9e1d191110e8399ef3eef44ef4a3a/lib/Model/ProjectVersion.php#L68-L72

Reply:
i copied it from https://github.com/doctrine/phpcr-odm/blob/2.x/.doctrine-project.json :sweat_smile: 

will remove it there as well